migrations/versions/0353_inbound_number_add_fields.py

```diff
@@ -0,0 +1,22 @@
+"""
+Revision ID: 0353_inbound_number_add_fields
+Revises: 0352_updated_at_default
+Create Date: 2022-11-07 19:30:54.448803
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+revision = '0353_inbound_number_add_fields'
+down_revision = '0352_updated_at_default'
+
+
+def upgrade():
+    op.add_column('inbound_numbers', sa.Column('url_endpoint', sa.String(), nullable=True))
+    op.add_column('inbound_numbers', sa.Column('self_managed', sa.Boolean(), nullable=False, server_default='0'))
```

Comment:
I've seen `f` and `false` in the database, why is this `'0'`?

Reply:
The [server_default parameter](https://docs.sqlalchemy.org/en/13/core/metadata.html?highlight=server_default#sqlalchemy.schema.Column.params.server_default) does not accept a boolean value.  I read somewhere, while troubleshooting a previous migration and in documentation I can't find now, that the strings '0' or '1' are portable.  Also see [here](https://docs.sqlalchemy.org/en/13/core/sqlelement.html#sqlalchemy.sql.expression.false).